extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end testing utilities for Algebraic Expression Input
+ * interaction
+ */
+
+var objects = require(process.cwd() + '/extensions/objects/protractor.js');
+
+var customizeInteraction = function() {
+  // There are no customizations.
+};
+
+var expectInteractionDetailsToMatch = async function(elem) {
+  expect(
+    await elem.element(by.tagName(
+      'oppia-interactive-algebraic-expression-input')).isPresent()
+  ).toBe(true);
+};
+
+var submitAnswer = async function(elem, answer) {
+  await objects.AlgebraicExpressionEditor(elem.element(by.tagName(
+    'oppia-interactive-algebraic-expression-input'))).setValue(answer);
```

Comment:
Can we also check that the edit is displayed to the editing user? This would help catch bugs where the edit is saved, but the editing user doesn't get feedback

Reply:
Again, assuming that we would have to fetch the current value of the editor, we can't, as mentioned in the [comment above](https://github.com/oppia/oppia/pull/9553#discussion_r444717025).